module_utils/k8s_common.py

```diff
@@ -162,12 +162,12 @@ def execute_module(self):
             if self.helper.base_model_name_snake.endswith('list'):
                 # For list modules, execute a GET, and exit
                 k8s_obj = self._read(name, namespace)
-                return_attributes[self.kind] = k8s_obj.to_dict() if k8s_obj else {}
```

Comment:
Were these removed on purpose? I think `k8s_obj` has a chance of being `None` which is why these guards were put in place.